### PR TITLE
Stabilize FormTitle component

### PR DESCRIPTION
**Pull request by jonkoops**

### CODE REVIEW

File: packages/asc-ui/src/components/Accordion/Accordion.stories.mdx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { withKnobs, boolean, text } from '@storybook/addon-knobs'
 import { Meta, Story, Preview } from '@storybook/addon-docs/blocks'
 import Accordion from './Accordion'
 import AccordionWrapper from './AccordionWrapper'
+import Paragraph from '../Paragraph'
```
Comment (robinpiets) [inline packages/asc-ui/src/components/Accordion/Accordion.stories.mdx:7]: Why import Paragraph here?

File: packages/asc-ui/src/components/FormTitle/FormTitle.stories.mdx [hunk 2]
```diff
@@ -22,23 +19,29 @@ export default {
...
     <FormTitle>This is the title of the FormTitle component</FormTitle>
-  </ThemeProvider>
-)
+  </Story>
+</Preview>
+
+## In context
 
-export const InContext = () => (
-  <ThemeProvider>
```
Comment (robinpiets) [inline packages/asc-ui/src/components/FormTitle/FormTitle.stories.mdx:41]: No need for ThemeProvider?

File: packages/asc-ui/src/components/FormTitle/FormTitle.stories.mdx [hunk 3]
```diff
@@ -22,23 +19,29 @@ export default {
...
+<Preview>
+  <Story name="In context">
     <Header homeLink="" tall />
     <FormTitle>This is the title of the FormTitle component</FormTitle>
     <Card maxWidth={500} backgroundColor="level2" shadow>
```
Comment (robinpiets) [inline packages/asc-ui/src/components/FormTitle/FormTitle.stories.mdx:47]: Somehow FormTitle and Card are not shown in Netlify preview in this Story

image

File: packages/asc-ui/src/components/Accordion/Accordion.stories.mdx (hunk 1 above)
Comment (jonkoops) [reply packages/asc-ui/src/components/Accordion/Accordion.stories.mdx:7]: It's used futher down, but the import was missing. Caused some errors to show up in the console.

File: packages/asc-ui/src/components/FormTitle/FormTitle.stories.mdx (hunk 3 above)
Comment (jonkoops) [reply packages/asc-ui/src/components/FormTitle/FormTitle.stories.mdx:47]: Yeah, haven't been able to figure this one out either. Looks like in all cases only the first component is rendered.

File: packages/asc-ui/src/components/FormTitle/FormTitle.stories.mdx (hunk 2 above)
Comment (jonkoops) [reply packages/asc-ui/src/components/FormTitle/FormTitle.stories.mdx:41]: Yeah, I am wondering if we need it in these examples. @chrisvanmook is it provided by default in the root somewhere?

Comment (chrisvanmook) [reply packages/asc-ui/src/components/FormTitle/FormTitle.stories.mdx:41]: @jonkoops its provided in `.storybook/config.js` as a decorator, so no need to use that here

Comment (jonkoops) [reply packages/asc-ui/src/components/FormTitle/FormTitle.stories.mdx:41]: Great, thanks for the clarification!

File: packages/asc-ui/src/components/FormTitle/FormTitle.stories.mdx (hunk 3 above)
Comment (robinpiets) [reply packages/asc-ui/src/components/FormTitle/FormTitle.stories.mdx]: image
Looks good now! Great

**Pull request merged**